docs/configuration.rst

```diff
@@ -472,3 +476,121 @@ Optional Settings
 All other settings for Connect like security, monitoring interceptors, producer and consumer overrides can be passed to the Docker images as environment variables. The names of these environment variables are derived by replacing ``.`` with ``_``, converting the resulting string to uppercase and prefixing it with ``CONNECT_``. For example, if you need to set ``ssl.key.password``, the environment variable name would be ``CONNECT_SSL_KEY_PASSWORD``.
 
 The image will then convert these environment variables to corresponding Connect config variables.
+
+------------------------------------------
+Confluent Enterprise Replicator Executable
+------------------------------------------
+
+Confluent Kafka Replicator Executable provides another way to run Replicator by consolidating configuration properties and abstracting Kafka Connect details. The image depends on input files that can be passed by mounting a directory with the expected input files or by mounting each file individually. Additionally, the image supports passing command line parameters to the Replicator executable via environment variables as well. For example:
+
+  .. sourcecode:: bash
+
+    docker run -d \
+      --name=ReplicatorX \
+      --net=host \
+      -e REPLICATOR_LOG4J_ROOT_LOGLEVEL=DEBUG \
+      -v /mnt/replicator/config:/etc/replicator \
+      confluentinc/cp-enterprise-replicator-executable:4.1.0
+
+will start Replicator given that the directory ``/mnt/replicator/config``, that will be mounted under ``/etc/replicator`` on the Docker image, contains the required files ``consumer.properties``, ``producer.properties`` and the optional but often necessary file ``replication.properties``.
```

Comment:
What happens if the directories are not properly mounted or the required files are not available?

Reply:
Nit: maybe say "given that the _local_ directory" (without emphasis) to help users avoid mounting incorrectly.

Reply:
I'd expect similar failures as when you don't set an environment variable. We'll explore such errors during testing and smoke testing of the images